### PR TITLE
Add workload-web-app repo to CI

### PR DESCRIPTION
**Pull request by STEPHINRACHEL**
**What**
Adding workoad-web-app repository to Openshift CI

### CODE REVIEW

**Comment by petr-muller** [comment]
/hold

If you want to act on the rehearse failure, otherwise feel free to unhold.

File: ci-operator/config/integr8ly/workload-web-app/integr8ly-workload-web-app-master.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: workload-web-app-base-image
+    namespace: integr8ly
+    tag: latest
```
Comment (mikenairn) [inline ci-operator/config/integr8ly/workload-web-app/integr8ly-workload-web-app-master.yaml:6]: Could you try:

```
base_images:
  os:
    cluster: https://api.ci.openshift.org
    name: centos
    namespace: openshift
    tag: "7"
build_root:
  image_stream_tag:
    cluster: https://api.ci.openshift.org
    name: release
    namespace: openshift
    tag: golang-1.13
```

File: ci-operator/config/integr8ly/workload-web-app/integr8ly-workload-web-app-master.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: Dockerfile
+  from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/integr8ly/workload-web-app
```
Comment (mikenairn) [inline ci-operator/config/integr8ly/workload-web-app/integr8ly-workload-web-app-master.yaml:20]: Remove this

**Comment by STEPHINRACHEL** [comment]
/cc @petr-muller 
Could you please add lgtm label?

**Pull request merged**